### DISCUSSION TITLE
Created a Discord Community (Unofficial)

### DISCUSSION THREAD

**Discussion by dylanh724** [sphinx-doc/sphinx#12402] [General]
[https://discord.gg/by9W5rKuKj](https://discord.gg/by9W5rKuKj)

Forums feel a bit dated and are slower to find answers, especially due to how particularly nuanced sphinx/rst is -- feel free to join and discuss in realtime in an unofficial Discord community!

Mods: Feel free to take it over later, if it grows and want it.

**Comment by chrisjsewell** [comment]
Heya yeh thanks, I was thinking to do this also because I'm certainly not a fan of the google mailing list, so am interested.

Obviously if it was to become "official" I feel we would want to do it in parallel with "closing down" the current google mail lists, and adding links in the documentation

One outstanding question I had was choosing between discord vs discourse (see e.g. https://www.commonroom.io/blog/discord-vs-slack-vs-discourse-as-your-community-channel/)
I think perhaps discord is better for real-time chat, but discourse may be better for (indexed) Q&A.
But then perhaps we would move to having this Github Discussions handle that use case 🤔 

I guess its just trying to get a balance between having the best communication platforms for each use case, but not having too many of them where it gets confusing

**Reply by dylanh724** [reply]
> Heya yeh thanks, I was thinking to do this also because I'm certainly not a fan of the google mailing list, so am interested.
> 
> Obviously if it was to become "official" I feel we would want to do it in parallel with "closing down" the current google mail lists, and adding links in the documentation
> 
> One outstanding question I had was choosing between discord vs discourse (see e.g. https://www.commonroom.io/blog/discord-vs-slack-vs-discourse-as-your-community-channel/) I think perhaps discord is better for real-time chat, but discourse may be better for (indexed) Q&A. But then perhaps we would move to having this Github Discussions handle that use case 🤔
> 
> I guess its just trying to get a balance between having the best communication platforms for each use case, but not having too many of them where it gets confusing

I've used (and hosted) both! Discourse is absolutely amazing, too, but... that realtime chat in 2024 is important. Folks want back-and-forth when discussing dev, not post and wait for a day with infinite email spam you eventually stop checking.

Discord recently filled the gap by providing optional threads for certain categories (eg: support requests). Google for OpenAI's Discord as an example.
